src/index.html

```diff
@@ -17,11 +17,452 @@
+        <a href="#">
+          <img
+            src="../src/images/logo.png"
+            alt="logo"
+            class="header__logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#apple"
+              class="nav__list-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#samsung"
+              class="nav__list-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#smartphones"
+              class="nav__list-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#laptops_and_computers"
+              data-qa="nav-hover"
+              class="nav__list-link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="# gadgets"
+              class="nav__list-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#tablets"
+              class="nav__list-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#photo"
+              class="nav__list-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#video"
+              class="nav__list-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac image"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-word">Price:</p>
+
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__btn-buy"
+          data-qa="hover"
```

Comment:
The `data-qa` attribute value should be unique if it's used for testing purposes. Here, `data-qa="hover"` is repeated multiple times which could lead to ambiguous test results.